lib/cashu/types.ts

```diff
@@ -0,0 +1,22 @@
+// [ "key", "value1", "value2", ...]
+export type NUT10SecretTag = [string, ...string[]];
+
+export const WELL_KNOWN_SECRET_KINDS = ['P2PK'] as const;
+export type WellKnownSecretKind = (typeof WELL_KNOWN_SECRET_KINDS)[number];
+
+export type NUT10SecretData = {
```

Comment:
looks good. I would maybe add some jsdoc for properties on these types with example values, e.g.:

```
type MoneyData = {
/**
 * <some explanation here>
 * @example '<string in the usual nonce format>'
 */
nonce: string
```

Reply:
added to all types